src/monit/Makefile

```diff
@@ -23,8 +23,21 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg init
 	stg import -s ../patch/series
 
+ifeq ($(CROSS_BUILD_ENVIRON), y)
+	# Copy monit configure settings for arm, add missing settings
+	cp /etc/dpkg-cross/cross-config.cache ./cross-config.cache
+	echo "libmonit_cv_setjmp_available=yes" >> ./cross-config.cache
+	echo "libmonit_cv_vsnprintf_c99_conformant=yes" >> ./cross-config.cache
+	sudo cp ./cross-config.cache /etc/dpkg-cross/cross-config.cache
```

Comment:
Hmm, is it possible for the monit build to not read from `/etc/dpkg-cross/cross-config.cache` and instead use a local file? This could, at worst, read to some race conditions when doing parallel builds.